packages/cli/src/controllers/oauth/oAuth1Credential.controller.ts

```diff
@@ -99,9 +99,8 @@ export class OAuth1CredentialController extends AbstractOAuthController {
 	}
 
 	/** Verify and store app code. Generate access tokens and store for respective credential */
-	@Get('/callback', { usesTemplates: true })
+	@Get('/callback', { usesTemplates: true, skipAuth: true })
```

Comment:
`skipAuth` also disables the browser ID checks?

Reply:
the browserId check happens in auth middleware. when `skipAuth` is `true`, that route does not use the auth middleware.